lib/gui/pages/settings/controllers/settings.js

```diff
@@ -71,6 +71,12 @@ module.exports = function(WarningModalService, SettingsModel) {
    */
   this.enableDangerousSetting = (name, options) => {
     if (!this.currentData[name]) {
+
+      AnalyticsService.logEvent('Set setting', {
+        setting: name,
+        value: false
+      });
```

Comment:
For symmetry with the other `'Set setting'` should this include `dangerous: false`? Or isn't that needed with the analytics service we're using?